cataclysmbnteam/Cataclysm-BN
# Update DeadPeople tileset

Update DeadPeople tileset



#### Summary
SUMMARY: Content  "Update DeadPeople tileset"


#### Purpose of change
Update DeadpPeople tileset


#### Describe the solution
Update tileset directory.
This time I've changed default TILES options to UNDEAD_PEOPLE_BASE. Game switched to it flawlessly and automatically - no need for migration of any kind. 
There is no need to change default tileset option in future - just bring new tileset directory.


#### Describe alternatives you've considered
None


#### Testing
1) Make sure that DeadPeople tileset selected.
2) Load game or start new one.
2) New Deadpeole tileset must be enabled.


#### Additional context
